nodejs/awsx/aws_test.go

```diff
@@ -225,21 +259,50 @@ func Test_Examples(t *testing.T) {
 					ExpectNoChanges: true,
 				},
 			},
-		}),
-		testBase.With(integration.ProgramTestOptions{
-			Dir:       path.Join(cwd, "../examples/alb/fargateInlineListener"),
+		})
+
+	integration.ProgramTest(t, &test)
+}
+
+func TestAccAlb_fargateInlineListener(t *testing.T) {
+	skipIfShort(t)
+	test := getBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:       path.Join(getCwd(t), "../examples/alb/fargateInlineListener"),
 			StackName: addRandomSuffix("fargate"),
-		}),
-		testBase.With(integration.ProgramTestOptions{
-			Dir:       path.Join(cwd, "../examples/alb/ec2Instance"),
+		})
+
+	integration.ProgramTest(t, &test)
+}
+
+func TestAccAlb_ec2Instance(t *testing.T) {
+	skipIfShort(t)
+	test := getBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:       path.Join(getCwd(t), "../examples/alb/ec2Instance"),
 			StackName: addRandomSuffix("ec2Instance"),
-		}),
-		testBase.With(integration.ProgramTestOptions{
-			Dir:       path.Join(cwd, "../examples/alb/lambdaTarget"),
+		})
+
+	integration.ProgramTest(t, &test)
+}
+
+func TestAccAlb_lambdaTarget(t *testing.T) {
+	skipIfShort(t)
+	test := getBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:       path.Join(getCwd(t), "../examples/alb/lambdaTarget"),
 			StackName: addRandomSuffix("lambdaTarget"),
-		}),
-		testBase.With(integration.ProgramTestOptions{
-			Dir:       path.Join(cwd, "../examples/nlb/fargate"),
+		})
+
+	integration.ProgramTest(t, &test)
+}
+
+func TestAccNlb_fargate(t *testing.T) {
+	skipIfShort(t)
+	envRegion := getEnvRegion(t)
+	test := getBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:       path.Join(getCwd(t), "../examples/nlb/fargate"),
```

Comment:
Nit: Since every test sets the directory `"../examples/"`, perhaps we should add the subfolder (e.g. `"nlb/fargate"`) as a second parameter to `getBaseOptions(t)`? That might clean things up a little bit, what do you think?

Reply:
Good suggestion here - I will follow up and run this across our repo as a separate PR when I get some cleanup time